packages/front-end/components/Experiment/ResultsDownloadButton.tsx

```diff
@@ -87,8 +87,8 @@ export default function ResultsDownloadButton({
             chanceToBeatControl: stats.chanceToWin || null,
             percentChange: stats.expected || null,
             percentChangePValue: stats.pValue || null,
-            percentChangeCILower: stats.ci[0] || null,
-            percentChangeCIUpper: stats.ci[1] || null,
+            percentChangeCILower: stats.ci?.[0] || null,
+            percentChangeCIUpper: stats.ci?.[1] || null,
```

Comment:
Wait, what's this bug (I introduced it)? Does it throw some error if `ci` is missing that I missed during my testing?

Reply:
ahhhh in the console